test/ha/ha.go

```diff
@@ -51,7 +51,7 @@ func extractDeployment(pod string) string {
 // GetLeaders collects all of the leader pods from the specified deployment.
 // GetLeaders will return duplicate pods by design.
 func GetLeaders(t *testing.T, client *test.KubeClient, deploymentName, namespace string) ([]string, error) {
```

Comment:
same re: breaking this API.

Reply:
done in efdb1fc3a7cff6eec4d060e3a5744446051894b5